ymir/command/mir/commands/importing.py

```diff
@@ -142,6 +142,9 @@ def _generate_sha_and_copy(index_file: str, sha_idx_file: str, sha_folder: str)
     with open(index_file) as idx_f, open(sha_idx_file, 'w') as sha_f:
         lines = idx_f.readlines()
         total_count = len(lines)
+        if total_count > 1000000:  # large number of images may trigger redis timeout error.
+            logging.error(f'# of image {total_count} exceeds upper boundary.')
```

Comment:
f'# of image {total_count} exceeds upper boundary 1000000

Reply:
done